deployments/operator/webhook/manifests.yaml

```diff
@@ -12,7 +12,7 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
- path: /mutate-deviceplugin-intel-com-v1-dlbdeviceplugin
+      path: /mutate-deviceplugin-intel-com-v1-dlbdeviceplugin
```

Comment:
Can you explain this change?

Reply:
it should be dropped from this PR via rebase. #756 fixed that typo

Reply:
Idk why it was put like that from the previous merged commit, but path should be there to make operator work..!
It was just wrong yaml grammar from the previous one.

Do you want me to make another separate pr for this?